wiki/Tournaments/HCΩ/en.md

```diff
@@ -0,0 +1,265 @@
+
+1. The schedule for the tournament is listed under [tournament scheduling](#tournament-schedule), please use the time in-between to practice the mappool(s).
+2. Matches are allowed to overlap if referees are available.
+3. For 1v1 Bracket:
+   - If both players can come up with a better time than what we the organizers has scheduled before, please inform the organizers as soon as possible so they can make adjustments as needed.
+4. Rescheduling is allowed but please give ample time to for tournament staff to figure out the logistics of the change as just because you told tournament staff, does **not** mean it is accepted.
+
+### Mappools
+
+1. For more specifics for what maps were played see: [Mappools](#mappools)
+2. There will be a variety of *Harumachi Clover* maps to choose from in each stage of the tournament.
+   - Note: 1v1 Bracket stage **does not** allow bans. See: [1v1 Bracket](#tournament-staging)
+3. Each stage will have the number of maps and mods as seen below:
+   - Qualifiers/Groups: 1NM, 1HD, 1HR, 1DT, 1FM
+   - RO16/QuarterFinal: 2NM, 1HD, 1HR, 1DT, 1FM, 1TB
+   - SemiFinal: 2NM, 1HD, 1HR, 2DT, 2FM, 1TB
+   - GrandFinal: 2NM, 2HD, 2HR, 2DT, 2FM, 1TB
+
+### Seeding specifics
+
+1. Seeding will take place during the Qualifiers and the Group Heat stages.
+2. During the Qualifiers Stage there will be multiple lobbies open for sign-ups. Please make sure to sign up for a time that best fits your schedule.
+3. The Qualifiers Stage is used to help get rid of those who don't show up and to get the participation count to the nice number of 64.
+4. There will be no warmups for the Qualifiers and Group Heats stages, remember this is a harumachi clover tournament, they are only 40 seconds at most.
+5. Seeding will be gathered by taking the highest score is recorded and used in comparison to other scores from the players.
+6. For the Qualifiers and Group Heats stage, **the mappool will be played through twice**.
+7. Players will be allowed to ask for a brief break at the end of the first run-through of the mappool.
+
+### Lobby specifics
+
+1. A private match will be created for the matches. The players will be invited for the asscociated match around 15 minutes before the match time. Make sure to have PMs enabled to everyone not just on your friends list as the invite does **not** bypass this setting. If you are unable to make it to the match due to this issue, your position in the tournament is **not** guaranteed.
+2. During the 1v1 Bracket, each player will `!roll`, the winner of the roll **picks who will pick the first map**. Tournament staff would like to preface, if you win the `!roll`, this does **not** mean you pick the first map, this means you pick whether you pick the first map or your opponent does. Choose wisely!
+3. There are **no** bans.
+4. There are **no** warmups. Be prepared.
+5. 1v1 Bracket only: In case of a tie, the tiebreaker will be played.
+   - Otherwise, see: [Number 6 of General Rulset](#ruleset).
+
+## History
```

Comment:
all below information should be in separate articles

Reply:
official support doesn't have any relation to tournament articles

Reply:
should be fixed as [#7244019](https://github.com/ppy/osu-wiki/pull/7551/commits/7244019a29693544fb4d59739f5d6eaafa446028)